### ISSUE TITLE
Link 'Learning Path' author to his public profile page in search results

### ISSUE THREAD

**Issue by xlogix** [fnplus/footsteps-app#64]
We recently added the ability to view the public profile of a user. All public user profiles are now visible at https://www.footsteps.dev/user/<your-username>. Now we want to link the public profile in the search results page so anyone can discover other learning paths by the same user.

Example Link: https://www.footsteps.dev/user/abhi
<img width="235" alt="Screenshot 2020-03-03 at 7 31 33 PM" src="https://user-images.githubusercontent.com/16256275/75783008-cf68b200-5d85-11ea-8fd9-d6ecb424cae6.png">

**Comment by imrishabh18** [comment]
Hey, I want to work on this issue.

**Comment by xlogix** [comment]
Assigned!

**Comment by Tanmayi03** [comment]
Is the issue still open? Can I work on this?

**Comment by xlogix** [comment]
Assigned!

**Comment by Tanmayi03** [comment]
@xlogix the feature is already present, right? The names are clickable and they point to the public profile of the user
![Screenshot (92)](https://user-images.githubusercontent.com/45629438/81491764-66cf1180-92af-11ea-9291-e71784fe86a9.png)

**Comment by xlogix** [comment]
Yes, but sometimes the name isn't clickable. It opens the card instead of going to the profile. Can you make it better? Maybe update a UI?

**Comment by Tanmayi03** [comment]
Yeah, I noticed that. I opens the card on clicking. On clicking it again, it's opening the profile. I'll try to make it better
